frontend/public/locales/fr/learn.json

```diff
@@ -3,70 +3,70 @@
+      "body": "Soyez plus confiant dans les choix que vous faites concernant votre retraite. Utilisez ces ressources pour mieux comprendre les options qui s'offrent à vous et leur incidence sur vos prestations.",
       "cards": [
         {
-          "title": "(FR) When to take government pensions",
-          "read": "(FR) 5 MIN READ",
-          "body": "(FR) Lorem ipsum dolor sit amet consectetur. Gravida eget tristique in ipsum eget ornare libero.",
+          "title": "Quand commencer à recevoir des pensions gouvernementales",
+          "read": "Temps de lecture : 9 minutes",
+          "body": "Vous pouvez choisir le moment où vous souhaitez commencer à recevoir vos prestations de vieillesse. Assurez-vous de bien comprendre les éléments dont il faut tenir compte pour prendre cette décision.",
           "image": "/assets/deciding-when-to-collect-public-pensions-sm.jpg",
           "link": "/learn/deciding-when-to-collect-public-pensions"
         },
         {
-          "title": "(FR) Going from work to retirement",
-          "read": "(FR) 7 MIN READ",
-          "body": "(FR) Lorem ipsum dolor sit amet consectetur. Gravida eget tristique in ipsum eget ornare libero.",
+          "title": "Working during retirement",
+          "read": "Temps de lecture: 7 minutes",
+          "body": "De nombreux Canadiens continuent à travailler pendant la retraite. Découvrez l'incidence que peuvent avoir les sommes que vous gagnez sur vos prestations.",
           "image": "/assets/going-from-work-to-retirement-sm.jpg",
           "link": "/learn/going-from-work-to-retirement"
         },
         {
-          "title": "(FR) Rules of thumb for public pensions",
-          "read": "(FR) 5 min read",
-          "body": "(FR) Pension tips based on your personal situation.",
+          "title": "Règles générales concernant les prestations de vieillesse fédérales",
+          "read": "Temps de lecture: 5 minutes",
+          "body": "Conseils en matière de pension selon votre situation personnelle.",
           "image": "/assets/rules-of-thumb-for-public-pensions-sm.jpg",
           "link": "/learn/rules-of-thumb-for-public-pensions"
         }
       ]
     },
     {
-      "title": "(FR) Federal benefit scenarios: from retirees to new pensioners",
-      "body": "(FR) Not sure when to retire? Learn about the different choices people make about their public pensions and how it impacts their lives. These stories give valuable lessons and inspiration for people looking to plan their own retirement.",
+      "title": "Histoires réelles de retraite",
+      "body": "Vous ne savez pas quand prendre votre retraite. Lisez les histoires dont certaines personnes nous ont fait part. Découvrez les différents choix et comparez-les à votre situation personnelle.",
       "cards": [
         {
           "title": "(FR) Fred decides when to take his public pensions",
-          "read": "(FR) 9 min read",
+          "read": "Temps de lecture: 9 minutes",
```

Comment:
```suggestion
          "read": "Temps de lecture\u00a0: 9 minutes",
```

Reply:
Thanks! Let me ask the content creators.